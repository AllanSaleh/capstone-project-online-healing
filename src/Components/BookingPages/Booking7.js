import React from 'react';

const Booking7 = () => (
  <div className="h-firstsection px-sides pt-navbar">
    <h1 className="text-title uppercase mb-4">Submit your appointment</h1>
    <h3 className="text-subtitle opacity-50 text-justify">
      Click Submit if you are sure of all your choices.
    </h3>
    <div className="flex flex-col justify-evenly items-center h-2/3 max-w-3xl my-16 mx-auto p-8 shadow-md">
      <h3 className="text-3xl capitalize">submit appointment?</h3>
      <p className="text-subtitle">
        Please be aware that this action will cost you a ticket!
      </p>
      <button
        type="button"
        className="uppercase bg-blue-dark text-subtitle px-4 py-2 min-w-min w-1/3 rounded-md border hover:border-blue-dark hover:bg-white hover:text-blue-dark transition-all duration-300"
      >
        submit
      </button>
    </div>
  </div>
);

export default Booking7;
