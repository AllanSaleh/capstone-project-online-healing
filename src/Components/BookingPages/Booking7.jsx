import React from 'react';
import { useTranslation } from 'react-i18next';
import { useHistory } from 'react-router-dom';

export default function Booking() {
  window.scrollTo(0, 0);
  const { t } = useTranslation();

  const history = useHistory();

  const PrevPage = () => {
    // Put firestore code here!
    history.push('/Booking6');
  };
  const NextPage = () => {
    // Put firestore code here!
    history.push('/Booking8');
  };
  return (
    <div className="lg:h-firstsection px-sides pt-navbar">
      <h1 className="text-2xl lg:text-title uppercase my-8">{t('Booking.Title3')}</h1>
      <h3 className="text-md lg:text-subtitle opacity-50 text-justify">{t('Booking.Subtitle3')}</h3>
      <div className="flex flex-col justify-evenly items-center h-1/2 lg:h-2/3 w-full lg:max-w-3xl my-16 mx-auto p-8 shadow-md">
        <h3 className="text-xl lg:text-3xl capitalize">{t('Booking.Submit')}</h3>
        <p className="text-sm lg:text-subtitle text-center">{t('Booking.Aware')}</p>
        <div className="flex justify-evenly mt-16 w-full">
          <button
            onClick={() => PrevPage()}
            type="button"
            className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition duration-300"
          >
            {t('Booking.Back')}
          </button>
          <button
            onClick={() => NextPage()}
            type="button"
            className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition-all duration-300"
          >
            {t('Booking.SubmitBtn')}
          </button>
        </div>
      </div>
    </div>
  );
}
