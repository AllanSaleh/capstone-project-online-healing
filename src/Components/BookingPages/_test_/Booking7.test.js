import * as React from 'react';
import * as renderer from 'react-test-renderer';
import Booking7 from '../Booking7';

it('Booking7 renders correctly', () => {
  const tree = renderer.create(<Booking7 />).toJSON();
  expect(tree).toMatchSnapshot();
});
