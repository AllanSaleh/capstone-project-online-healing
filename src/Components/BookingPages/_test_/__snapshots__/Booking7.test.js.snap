// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Booking7 renders correctly 1`] = `
<div
  className="h-firstsection px-sides pt-navbar"
>
  <h1
    className="text-2xl lg:text-title uppercase my-8"
  >
    Submit your appointment
  </h1>
  <h3
    className="text-md lg:text-subtitle opacity-50 text-justify"
  >
    Click Submit if you are sure of all your choices.
  </h3>
  <div
    className="flex flex-col justify-evenly items-center h-1/2 lg:h-2/3 w-full lg:max-w-3xl my-16 mx-auto p-8 shadow-md"
  >
    <h3
      className="text-xl lg:text-3xl capitalize"
    >
      submit appointment?
    </h3>
    <p
      className="text-sm lg:text-subtitle text-center"
    >
      Please be aware that this action will cost you a ticket!
    </p>
    <div
      className="flex justify-evenly mt-16 w-full"
    >
      <button
        className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition duration-300"
        onClick={[Function]}
        type="button"
      >
        Back
      </button>
      <button
        className="uppercase bg-blue-dark h-12 w-32 text-md lg:text-subtitle rounded-lg border hover:bg-transparent hover:border-blue-dark hover:text-blue-dark transition-all duration-300"
        onClick={[Function]}
        type="button"
      >
        Submit
      </button>
    </div>
  </div>
</div>
`;
